src/vcf-autosuggest.js

```diff
@@ -201,7 +201,9 @@ import './vcf-autosuggest-overlay';
     }
 
     attached() {
-        this._defaultOptionChanged(this.defaultOption);
+        super.attached();
+        if (this._hasDefaultOption())
```

Comment:
This check is important, baecause otherwise it looses the old value in case the component is reused